modules/cluster-csi-snapshot-controller-operator.adoc

```diff
@@ -1,16 +1,54 @@
 // Module included in the following assemblies:
 //
 // * operators/operator-reference.adoc
+// * installing/cluster-capabilities.adoc
 
+ifeval::["{context}" == "cluster-operators-ref"]
+:operator-ref:
+endif::[]
+
+ifeval::["{context}" == "cluster-capabilities"]
+:cluster-caps:
+endif::[]
+
+:_content-type: REFERENCE
 [id="cluster-csi-snapshot-controller-operator_{context}"]
-= Cluster CSI Snapshot Controller Operator
+ifdef::operator-ref[= Cluster CSI Snapshot Controller Operator]
+ifdef::cluster-caps[= CSI snapshot controller capability]
+
+ifdef::operator-ref[]
+
+[NOTE]
+====
+The Cluster CSI Snapshot Controller Operator is an optional cluster capability that can be disabled by cluster administrators during installation. For more information about optional cluster capabilities, see "Cluster capabilities" in _Installing_.
```

Comment:
Is the link to installing assuming most readers will read the operator-ref location before they install a cluster (I would expect most readers to find it after having installed a cluster)?  Or is it because the installing docs are considered the main entry-point, with install->day-2 linking somehow being easier to follow than day-2->install linking?  Or some other reason?

Reply:
The reason is so that if the customer finds their way to this page from anywhere *except* the install caps page (such as googling the operator name), they are made aware that there's a caps page detailing how to toggle the cap.

Reply:
but "how to toggle the cap" depends on whether you have a cluster (in which case you want the day 2 page (`post_installation_configuration/enabling-cluster-capabilities.html`) or not (in which case you want the install page (`installing/cluster-capabilities.html`), so it seems like we'd either need to make that distinction here with pointers to both locations, or pick a favorite here, and then ensure cross linking between install and day-2 to handle folks where our favorite was wrong.

Reply:
I see what you mean. The cluster capabilities page in the Install section does have a link to the post-install "enabling capabilities" page as well, so both cases should be covered by the one link.